test/e2e/http2.test.js

```diff
@@ -1,238 +0,0 @@
-          http2: true,
-          port,
-        },
-        compiler
-      );
-
-      await server.start();
-
-      ({ page, browser } = await runBrowser());
-
-      pageErrors = [];
-      consoleMessages = [];
-    });
-
-    afterEach(async () => {
-      utilSpy.mockRestore();
-      await browser.close();
-      await server.stop();
-    });
-
-    it("should handle GET request to index route (/)", async () => {
-      page
-        .on("console", (message) => {
-          consoleMessages.push(message);
-        })
-        .on("pageerror", (error) => {
-          pageErrors.push(error);
-        });
-
-      const response = await page.goto(`https://127.0.0.1:${port}/`, {
-        waitUntil: "networkidle0",
-      });
-
-      const HTTPVersion = await page.evaluate(
-        () => performance.getEntries()[0].nextHopProtocol
-      );
-
-      expect(utilSpy.mock.calls[0][1]).toBe(
-        "'http2' option is deprecated. Please use the 'server' option."
-      );
-
-      expect(HTTPVersion).toMatchSnapshot("HTTP version");
-
-      expect(response.status()).toMatchSnapshot("response status");
-
-      expect(await response.text()).toMatchSnapshot("response text");
-
-      expect(consoleMessages.map((message) => message.text())).toMatchSnapshot(
-        "console messages"
-      );
-
-      expect(pageErrors).toMatchSnapshot("page errors");
-    });
-  });
-
-  describe("https without http2, disables HTTP/2", () => {
-    let compiler;
-    let server;
-    let page;
-    let browser;
-    let pageErrors;
-    let consoleMessages;
-
-    beforeEach(async () => {
-      compiler = webpack(config);
-
-      server = new Server(
-        {
-          static: staticDirectory,
-          https: true,
-          http2: false,
-          port,
-        },
-        compiler
-      );
-
-      await server.start();
-
-      ({ page, browser } = await runBrowser());
-
-      pageErrors = [];
-      consoleMessages = [];
-    });
-
-    afterEach(async () => {
-      await browser.close();
-      await server.stop();
-    });
-
-    it("should handle GET request to index route (/)", async () => {
-      page
-        .on("console", (message) => {
-          consoleMessages.push(message);
-        })
-        .on("pageerror", (error) => {
-          pageErrors.push(error);
-        });
-
-      const response = await page.goto(`https://127.0.0.1:${port}/`, {
-        waitUntil: "networkidle0",
-      });
-
-      const HTTPVersion = await page.evaluate(
-        () => performance.getEntries()[0].nextHopProtocol
-      );
-
-      expect(HTTPVersion).toMatchSnapshot("HTTP version");
-
-      expect(response.status()).toMatchSnapshot("response status");
-
-      expect(await response.text()).toMatchSnapshot("response text");
-
-      expect(consoleMessages.map((message) => message.text())).toMatchSnapshot(
-        "console messages"
-      );
-
-      expect(pageErrors).toMatchSnapshot("page errors");
-    });
-  });
-});
```

Comment:
Just for check - do we have the same test cases for the other options (same for `https`)

Reply:
Yes, all tests cases are present for the `server` option.